velox/functions/prestosql/window/tests/WindowTestBase.h

```diff
@@ -109,50 +116,45 @@ class WindowTestBase : public exec::test::OperatorTestBase {
     velox::window::prestosql::registerAllWindowFunctions();
   }
 
-  /// This function generates a simple two integer column RowVector for tests.
-  /// The first integer column has row number % 5 values.
-  /// The second integer column has row number % 7 values.
+  /// The below 4 functions are used to generate input data vectors for window
+  /// function testing. The vectors have 4 columns:
+  /// The first two columns are used to test different partition and sorting
+  /// combinations in the over clause.
+  /// The third and fourth columns are used to test frame clauses. So they
+  /// contain positive non-zero values.
+
+  /// This function generates uniformly distributed partition and sort values.
   RowVectorPtr makeSimpleVector(vector_size_t size);
 
-  /// This function generates a two integer column RowVector for tests.
-  /// The intention here is that the first column has a constant value of 1.
-  /// The second column has a value of the row number.
-  /// This tests the case where all data is in a single partition.
+  /// This function generates data where all rows belong to a single partition.
   RowVectorPtr makeSinglePartitionVector(vector_size_t size);
 
-  /// This function generates a two integer column RowVector for tests.
-  /// Both the first and second column of each data row is the row number.
+  /// This function generates data where all partitions have a single row.
   RowVectorPtr makeSingleRowPartitionsVector(vector_size_t size);
 
-  /// This function generates test data using the VectorFuzzer.
-  std::vector<RowVectorPtr> makeFuzzVectors(
-      const RowTypePtr& rowType,
-      vector_size_t size,
-      int numVectors,
-      float nullRatio = 0.0);
-
-  /// This function generates a column of test data using the VectorFuzzer.
-  VectorPtr makeFlatFuzzVector(
-      const TypePtr& type,
-      vector_size_t size,
-      float nullRatio = 0.0);
-
-  /// Helper function to generate a list of over clauses by appending a suffix
-  /// to the input list. Used to get over clauses containing all input columns
-  /// in order to impose a deterministic output row order.
-  std::vector<std::string> addSuffixToClauses(
-      const std::string& suffix,
-      const std::vector<std::string>& inputClauses);
+  /// This function generates a vector of random input values.
+  RowVectorPtr makeRandomInputVector(vector_size_t size);
+
+  VectorPtr
+  makeFlatFuzzVector(const TypePtr& type, vector_size_t size, float nullRatio);
```

Comment:
consistency: makeRandomInputVector vs. makeFlatFuzzVector

Consider renaming makeFlatFuzzVector to makeRandomFlatVector.